resolwe_bio/tests/test_processor.py

```diff
@@ -1,12 +1,15 @@
 # pylint: disable=missing-docstring
 from django.test import TestCase
 
-from server.models import Processor, iterate_schema
+try:
```

Comment:
I do not like this. I suggest to implement a ```get_processes``` method in utils or a similar solution. Utils.py should be the only file with references to genesis, so that we have more control and that it will be easy to remove it.